torax/tests/test_data/test_prescribed_particle_source.py

```diff
@@ -0,0 +1,51 @@
+"""Tests setting an externally-prescribed, time-evolving particle source.
+
+Notable settings:
+- Constant transport coefficient model
+- Pedestal
+"""
+
+CONFIG = {
+    'runtime_params': {
+        'profile_conditions': {
+            'set_pedestal': True,
+            'nbar': 0.85,
+        },
+        'numerics': {
+            'ion_heat_eq': True,
+            'el_heat_eq': True,
+            'dens_eq': True,
+            'current_eq': True,
+            't_final': 5,
+        },
+    },
+    'geometry': {
+        'geometry_type': 'circular',
+    },
+    'sources': {
+        'generic_ion_el_heat_source': {},
+        'qei_source': {},
+        'gas_puff_source': {
+            'S_puff_tot': 1.0e22,
+        },
+        'pellet_source': {
+            'S_pellet_tot': {0: 5.0e22, 2.5: 5.0e22, 2.6: 0.5e22},
```

Comment:
I'm not sure that this is testing the new PRESCRIBED mode, since this uses preexisting features

Reply:
You're absolutely right, I'll change it.

Reply:
Changed to prescribing jext in 0c8d5e693db052bc3c475fa0dab00613c90e4800